app/javascript/packs/OAS3Autocomplete.js

```diff
@@ -88,15 +91,7 @@ const injectServerToResponseBody = (responseBody: ResponseBody | string, service
   }
 }
 
-// response.body.method is not present when fetching the spec,
-// is present when doing a request to one of the paths
-const isSpecFetched = (response: SwaggerResponse): boolean => !!response.body.method
```

Comment:
I just realized this method is totally useless, don't remember why it was included